### DISCUSSION TITLE
Multi-Tool doesn't function as a wrench properly

### DISCUSSION THREAD

**Discussion by mijukamidami22** [FakeFishGames/Barotrauma#15050] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When using the multitool anything that requires a wrench to be done (not repaired) will not be done.
The most clear example is detaching anything that was placed, such as buttons or components.
I think an easy fix is to add a toggle to the multitool for when it will act as a wrench and when as a screwdriver, perhaps with m1 and m2? or other keys as well.
I do want to reiterate repairing works fine, i have tested it on every part of the ship including but not limited to:
Gun loaders, pumps, turbine, fabricator, deconstructor, medical fabricator, electrical boxes, battery chargers, reactor, oxygen generator, supercapacitors.

### Reproduction steps

I Start any campaign or go into the sub editor,
II Spawn in a button or any component and the multi-tool
III Place said component or button and try to remove it with the multi tool

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by pavel4win** [comment]
Wiki says that the multi-tool is designed for repair, NOT for components detaching.
However, the multi-tool disconnects detonators.

**Comment by Regalis11** [comment]
Thank you for the report! It is actually intentional that the multitool doesn't detach components, since that'd make the repair action conflict with the detaching action. We've now addressed this in our private repo by making the multitool not detach any item (such as the detonators mentioned above) to keep things consistent.

**Reply by Antopsy** [reply]
Right. So there IS a problem with consistency here. If player is holding both multitool and a wrench it still conflicts with each other. I think you really need to solve this issue if you want to use multitool as an instrument because as of now it does conflict with the main mechanics of the game.
